src/ffi.rs

```diff
@@ -21,8 +21,9 @@ use rand_core::{
     impls::{next_u32_via_fill, next_u64_via_fill},
     CryptoRng, RngCore,
 };
+use rusk_abi::POSEIDON_TREE_DEPTH;
```

Comment:
I'm totally fine if you want to re-export it btw, so in the tests you don't need to use `rusk-abi` only for it.
Just use `pub use` (my comment wasn't about the re-exporting much as the way you were doing it.)